configs/linode.json

```
@@ -0,0 +1,18 @@
+{
+    "provider": "linode",
+    "description": "Linode estimates based on July 2024 advertised prices",
+    "linodeTokenSecret": "kube-system/linode",
+    "CPU": "0.0",
+    "spotCPU": "0.0",
+    "RAM": "0.0",
+    "spotRAM": "0.0",
+    "GPU": "0.0",
+    "spotGPU": "0.0",
```

We can estimate the values using e.g. Linear Regression, should be good enough for fallback values:

```suggestion
    "CPU": "0.03",
    "spotCPU": "0.0",
     "RAM": "0.01",
     "spotRAM": "0.0",
     "GPU": "0.67",
     "spotGPU": "0.0",
```